src/MaxText/configs/base.yml

```diff
@@ -860,3 +860,6 @@ subslice_shape: ""
 # NNX
 enable_nnx: false
 shard_optimizer_over_data: False
+
+# Use tokamax library for kernel implementations
+use_tokamax: false
```

Comment:
It might be good to distinguish specific tokamax kernels, `use_tokamax_gmm` `use_tokamax_splash` etc